packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx

```diff
@@ -513,15 +638,66 @@ export class ViewerTransformWidget {
 
     window.addEventListener('pointermove', this.handlePointerMove);
 
-    this.getTransformWidget().updateDisabledAxis({
-      xRotation: this.xRotationDisabled,
-      yRotation: this.yRotationDisabled,
-      zRotation: this.zRotationDisabled,
+    this.completeEndTransform();
+  };
 
-      xTranslation: this.xTranslationDisabled,
-      yTranslation: this.yTranslationDisabled,
-      zTranslation: this.zTranslationDisabled,
-    });
+  private handleInputIncrement = (): void => {
+    if (this.inputValue != null && this.lastInputValue != null) {
+      this.inputValue = this.lastInputValue + 1;
+      if (this.isModifyingAngleUnits()) {
+        this.inputValue = this.inputValue % 360;
```

Comment:
Do we know that this.inputValue is a positive number? I know in other places we do `(degrees + 3600) % 360`

Reply:
Yeah, in the case of modifying angle units we'll always have a positive value here, since we do this check when the angle goes below zero. However, I hadn't realized we had the `Angle.normalize()` utility with the logic you're describing to do exactly what I'm trying to do here, so I'll remove this and use that utility!